lib/src/base.dart

```diff
@@ -121,12 +121,12 @@ abstract class SentryClient {
 
   @visibleForTesting
   String get postUri {
-    String port = dsnUri.hasPort &&
+    var port = dsnUri.hasPort &&
```

Comment:
Here and elsewhere, if something is assigned only once, consider making it `final`.

Reply:
Actually, nvm. I'll merge. Let's fix this with a lint later.

Reply:
Got it. Thanks for clarifying.
I'll add linting rules from flutter/flutter next